src/PanelTraits/Boxes.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Boxes
+{
+    public $sideBoxesEnabled = false;
+    public $boxesOptions = [];
+
+    public function enableSideBoxes()
+    {
+        $this->sideBoxesEnabled = true;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function disableSideBoxes()
+    {
+        $this->sideBoxesEnabled = false;
+
+        return $this->sideBoxesEnabled;
+    }
+
+    public function sideBoxesEnabled()
+    {
+        return $this->sideBoxesEnabled;
+    }
+
+    public function sideBoxesDisabled()
+    {
+        return ! $this->sideBoxesEnabled;
+    }
+
+    public function boxExists($label)
+    {
+        $boxes = $this->getBoxes();
+
+        return in_array($label, $boxes);
+    }
+
+    public function getLastBox()
+    {
+        $boxes = $this->getBoxes();
+
+        if (count($boxes)) {
+            return last($boxes);
+        }
+
+        return false;
```

Comment:
Backwards compatibility is important (although I suspect most could wouldn't do a `===` check against false).

Maybe we might suggest changing the method signatures in a future release.